src/main/java/com/openlattice/chronicle/configuration/TwilioConfiguration.kt

```diff
@@ -0,0 +1,38 @@
+package com.openlattice.chronicle.configuration
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.kryptnostic.rhizome.configuration.Configuration
+import com.kryptnostic.rhizome.configuration.ConfigurationKey
+import com.kryptnostic.rhizome.configuration.SimpleConfigurationKey
+import com.kryptnostic.rhizome.configuration.annotation.ReloadableConfiguration
+
+@ReloadableConfiguration(uri = "twilio.yaml")
+class TwilioConfiguration @JsonCreator constructor(
+    @get:JsonProperty(SID_PROPERTY)
+    @param:JsonProperty(SID_PROPERTY) val sid: String,
+    @get:JsonProperty(TOKEN_PROPERTY)
+    @param:JsonProperty(TOKEN_PROPERTY) val token: String,
+    @get:JsonProperty(FROM_PHONE)
+    @param:JsonProperty(FROM_PHONE) val fromPhone: String
```

Comment:
Seems like TwilioConfiguration class already exists: https://github.com/openlattice/openlattice/blob/main/conductor-client/src/main/java/com/openlattice/twilio/TwilioConfiguration.java

Reply:
there is no fromPhone in the TwilioConfig from conductor-client - since this will require changes there too, let's save this for another PR.